presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraDistributedQueries.java

```diff
@@ -150,16 +150,6 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Cassandra connector does not support column default values");
     }
 
-    @Override
-    public void testColumnName(String columnName)
```

Comment:
> Enable testColumnName in TestCassandraDistributedQueries


Consider squashing this commit with `Use Metadata.quote in CassandraCqlUtils` -- if this is what fixes Cassandra and makes this test pass now.